incubator/external-dns/values.yaml

```diff
@@ -0,0 +1,68 @@
+
+# Details about the image to be pulled.
+image:
+  repository: registry.opensource.zalan.do
+  name: teapot/external-dns
+  tag: v0.3.0
+  pullPolicy: IfNotPresent
+
+# This controls which types of resource external-dns should 'watch' for new
+# DNS entries.
+source:
+  fake: false
+  ingress: true
+  service: true
+
+# The Kubernetes API server to connect to, leave unset to use auto-detect.
+# master: http://127.0.0.1:8080
+
+# Limit sources of endpoints to a specific namespace.
+# namespace: all
+
+# A templated string that's used to generate DNS names
+# fqdnTemplate: {{.Name}}.my-org.com
+
+# Process annotation semantics from legacy implementations (options: mate, molecule )
+# compatibility: 
+
+# The DNS provider where the DNS records will be created (options: aws, google, inmemory, azure )
+provider: aws
+
+# When using the Google provider, specify the Google project (required when --provider=google)
+# googleProject:
+
+# Limit possible target zones by a domain suffix (optional)
+# domainFilter:
+
+# When using the Azure provider, override the Azure resource group to use (optional)
+# azureResourceGroup:
+
+# Modify how DNS records are sychronized between sources and providers (options: sync, upsert-only )
+policy: upsert-only
```

Comment:
My (slight) preference here is to explicitly set it, that prevents a situation where a user bumps the version of `external-dns` which may change it's default in the future meaning the state the chart leaves the service is would be unknown, i.e. a user would have to go and read the `external-dns` docs to discover the default for *their* chosen version.  I'm not hugely tided to that view though :)